src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/RunbookJobTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Commands.Automation.Test
+{
+    using Microsoft.Azure.Commands.Automation.Test;
+    using Microsoft.Azure.ServiceManagemenet.Common.Models;
+    using Microsoft.WindowsAzure.Commands.ScenarioTest;
+    using Xunit;
+
+    public class RunbookJobTests : AutomationScenarioTestsBase
+    {
+        public XunitTracingInterceptor _logger;
+
+        public RunbookJobTests(Xunit.Abstractions.ITestOutputHelper output)
+        {
+            _logger = new XunitTracingInterceptor(output);
+            XunitTracingInterceptor.AddToContext(_logger);
+        }
+
+        [Fact]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        public void TestCreateRunbookGraph()
+        {
+            // Write PS Function and call it here
+            RunPowerShellTest(_logger, "Test-CreateRunbookGraph -Name RB-Graphical-01");
+        }
+
+        [Fact]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        public void TestImportRunbookPowerShell()
+        {
+            RunPowerShellTest(_logger, "Test-ImportRunbookPowerShell -Name TestRunbook-PowerShellScript -RunbookPath ScenarioTests\\Resources\\RB-PowerShellScriptTutorial.ps1");
```

Comment:
The `RunbookPath` provided here will not work on Linux. Please use something like:
```cs
$"Test-ImportRunbookPowerShell -Name TestRunbook-PowerShellScript -RunbookPath {Path.Combine("ScenarioTests", "Resources", "RB-PowerShellScriptTutorial.ps1")}"
```